types.d.ts

```diff
@@ -11,6 +11,7 @@ export interface EnvironmentData {
 export interface RunOptions {
 	workspace: string
 	root: string
+	overrideRoot?: string
```

Comment:
This doesn't seem necessary. PNPM automatically applies the overrides from the workspace root to all sub-packages.

Reply:
Done